lms/djangoapps/ccx/api/v0/tests/test_views.py

```diff
@@ -1093,7 +1093,7 @@ def test_patch_detail_modules(self):
         resp = self.client.patch(self.detail_url, data, format='json', HTTP_AUTHORIZATION=self.auth)
         assert resp.status_code == status.HTTP_204_NO_CONTENT
         ccx_from_db = CustomCourseForEdX.objects.get(id=self.ccx.id)
-        six.assertCountEqual(self, ccx_from_db.structure, chapters)
+        assert ccx_from_db.structure == chapters
```

Comment:
same as #26502 , make sure you've resolved conflicts properly

Reply:
pyupgrade changed the `six.assertCountEqual` to `self.assertCountEqual` which I then manually changed to `assert`.